Jenkinsfile

```
@@ -101,6 +101,17 @@ pipeline {
               }
             }
           }
+          post {
+            always {
+              sh '''
+                rm -rf k-distribution/k-tutorial/1_basic/build
+                rm -rf kframework-${env.VERSION}
```

```suggestion
                rm -rf kframework-${VERSION}
```

Inside groovy scripts, you have to use the `${NAMESPACE.VARNAME}` for substitutions. But in bash blocks, the varibales `${env.VARNAME}` are exposed as environment variables.